pkg/instance/service.go

```diff
@@ -374,11 +374,16 @@ func (s service) findParameterValue(parameter string, sourceInstance *model.Inst
 	}
 
 	stackParameter, ok := sourceStack.Parameters[parameter]
-	if !ok {
-		return "", fmt.Errorf("unable to find value for parameter: %s", parameter)
+	if ok {
+		return *stackParameter.DefaultValue, nil
 	}
 
-	return *stackParameter.DefaultValue, nil
+	// TODO: Remove HostnamePattern once stacks 2.0 are the default
+	if parameter == "DATABASE_HOSTNAME" {
```

Comment:
Since you have the stack parameter you should be able to call the provider function without having to add the field HostnamePattern to the stack. I think 😋